loco-cli/src/bin/main.rs

```diff
@@ -24,8 +29,21 @@ enum Commands {
 fn main() -> eyre::Result<()> {
     let cli = Cli::parse();
 
+    println!("{:?}", &cli.verbose);
```

Comment:
you probably want to remove this right?

Reply:
Y, sure. One more question - will this change break a test? Probably I should fix it also

Reply:
@Virviil, you are correct. See [here](https://github.com/loco-rs/loco/blob/master/loco-cli/tests/cli_tests.rs#L3)  it is commented out. 
Do you mind to fix it?

Reply:
Fixed

Reply:
@kaplanelad I'm not sure that i'll take this task, but for sure not as a part of this PR. So let's just proceed